flexmeasures_client.py

```diff
@@ -83,11 +82,12 @@ def check_deprecation_and_sunset(self, url, res):
         ---------
         https://flexmeasures.readthedocs.io/en/latest/api/introduction.html#deprecation-and-sunset
         """
-        warnings = res.headers.get_all("Deprecation") + res.headers.get_all("Sunset")
+        self.log(res.headers)
```

Comment:
Should we really always log all headers here?

Reply:
Thanks, indeed not.